src/Main.js

```diff
@@ -4,30 +4,31 @@ import ExtendedView from './ExtendedView'
 import List from './List'
 import movieData from './mockData'
 import './Main.css'
+import {getAllMovies, fetchSingleMoviePoster} from './apiCalls'
 
 class Main extends React.Component {
   constructor() {
     super();
     this.state = {
       movieData: [],
+      singleMovie: [],
```

Comment:
I wonder if `movies` and `movie` would be more straightforward names for these properties?